### ISSUE TITLE
Bug: Folder exclusion does not work

### ISSUE THREAD

**Issue by DavidFarago** [alangrainger/obsidian-frontmatter-modified-date#26]
I have added my folder `templates` under "Exclude folders" in the configuration, but modified fields are still added to my notes under `templates`.

I am using the current version, 1.3.2.

**Comment by alangrainger** [comment]
Can you please test in a Sandbox vault? I can't replicate the issue, so it might be a conflict with another plugin which is causing this.

**Comment by DavidFarago** [comment]
Thanks for the fast reply. I tried in sandbox and it works:
* notes within the directory that I specified to exclude are NOT updated with the date metadata
* notes outside of the directory that I specified to exclude are updated with the date metadata.

I have the following plugins, besides yours, installed in my vault.
![Screenshot from 2024-06-15 10-46-03](https://github.com/alangrainger/obsidian-frontmatter-modified-date/assets/1004626/94ef6481-2024-4b11-b759-d7ffe094600f)

**Comment by alangrainger** [comment]
Perhaps just check also if the setting has taken effect after a restart. It could be simply that Obsidian needed to reload for it to properly work.

Otherwise, can you go through the plugins one-by-one to identify which is causing it to fail?

**Comment by DavidFarago** [comment]
The bug no longer occurs, thanks @alangrainger for your help.

I do not know what caused the problem, or how I fixed it:
* maybe I wrote a wrong directory name (I am pretty sure I tested it with several correctly written directory names and the bug occurred). Now it works, even for emojis in the directory name
*maybe I did forgot to restart obsidian (I am pretty sure I did)
* I deactivated all of the above plugins, except for git, and the bug disappeared. After re-activating all plugins one-by-one, the bug never re-occured. 
Thus I am closing this issue.
